R/Utilities.R

```diff
@@ -546,6 +551,45 @@ TADA_OrderCols <- function(.data){
   
 }
 
+#' Substitute Preferred Characteristic Name for Deprecated Names
+#' 
+#' This utility function uses the WQX Characteristic domain table to substitute
+#' deprecated characteristic names with the new name in the
+#' TADA.CharacteristicName column. Used in TADA_Autoclean function on data
+#' pulled from the WQP.
```

Comment:
TADA_SubstituteDeprecatedChars is run within TADA_Autoclean, which runs within TADA_DataRetreival and TADA_BigDataRetrieval. Therefore, deprecated characteristic names are harmonized to their current name automatically upon data retrieval. TADA_SubstituteDeprecatedChars can also be used by itself on a user supplied dataset that is in the WQX format, if desired.